tests/e2e/testRunner.ts

```diff
@@ -88,36 +86,39 @@ const runTests = async () => {
     await server.start();
 
     // Create a dict in which we will store the run durations for all tests
-    const results = {};
+    const results: Record<string, Record<string, unknown[]>> = {};
```

Comment:
What if we make it a little bit more readable, also I think we can use number[] instead of unknown[], wdyt?

```
type Result = Record<string, number[]>
...
const results: Record<string, Result> = {};
```

Reply:
Great approach, thanks!